src/commands/workflow/run.ts

```diff
@@ -0,0 +1,48 @@
+import { Flags } from "@oclif/core";
+
+import * as ApiV1 from "@/lib/api-v1";
+import BaseCommand from "@/lib/base-command";
+import { commaSeparatedStr, jsonStr } from "@/lib/helpers/flag";
+import { withSpinner } from "@/lib/helpers/request";
+import { indentString } from "@/lib/helpers/string";
+
+export default class WorkflowRun extends BaseCommand {
+  static flags = {
+    environment: Flags.string({
+      default: "development",
+      summary:
+        "The environment in which to run the workflow; defaults to development.",
+    }),
+    recipients: commaSeparatedStr({
+      required: true,
+      aliases: ["recipient"],
+      summary:
+        "A comma-separated list of user IDs that will receive this workflow. Max number of recipients is 5.",
+    }),
+    actor: Flags.string({ summary: "An optional actor for the workflow run." }),
+    tenant: Flags.string({
+      summary: "An optional tenant for the workflow run.",
+    }),
+    data: jsonStr({
+      summary: "An optional JSON string of the data for this workflow",
```

Comment:
```suggestion
      summary: "A JSON string of the data for this workflow",
```
I removed the "optional" bit here and in other places, because Oclif documents required vs optional flags automatically.